### PR TITLE
added email, phone, website fields to profile

### PR DESCRIPTION
**Pull request by Julietbc**

### CODE REVIEW

File: agents/components/Profile.js [hunk 1]
```diff
@@ -70,6 +70,48 @@ function Profile (props) {
...
+          name: 'email',
+          floatingLabelText: (
+            h(FormattedMessage, {
+              id: 'agents.email',
+              className: styles.labelText
+            })
+          ),
+          component: TextField,
+          fullWidth: true,
+          multiLine: true,
```
Comment (ahdinosaur) [inline agents/components/Profile.js]: why `multiLine`?

Comment (ahdinosaur) [reply agents/components/Profile.js]: i guess this question goes for all the new fields.

File: agents/schemas/profile.json [hunk 2]
```diff
@@ -26,6 +26,18 @@
       "faker": "internet.avatar",
       "format": "uri",
       "description": "Agent avatar image link"
+    },
+    "email" : {
+      "type": "string",
```
Comment (ahdinosaur) [inline agents/schemas/profile.json]: we could add format email for this.

**Review by ahdinosaur — COMMENTED** [review]
rubble dubble yub wub

**Review by iainkirkpatrick — APPROVED** [review]
think it's good for now - we can add the markdown editor later if we want, maybe more advanced schema stuff for fields as well. but doth demo waits for no human ;)

File: agents/schemas/profile.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
       "description": "Agent avatar image link"
     },
     "email" : {
-      "type": "string",
+      "type": "emails",
```
Comment (ahdinosaur) [inline agents/schemas/profile.json]: hey @iainkirkpatrick, i don't think this is legit JSON Schema.

i'd expect `"type": "string"`

and then another line `"format": "email"`.

or am i missing some special `emails` type?

Comment (iainkirkpatrick) [reply agents/schemas/profile.json]: ooooh my bad! good catch - just checked and i totally read the schema docs wrong! will fix with fix branch

**Pull request merged**